src/game.cpp

```diff
@@ -4004,12 +4006,7 @@ bool Game::combatChangeHealth(Creature* attacker, Creature* target, CombatDamage
 		realHealthChange = target->getHealth() - realHealthChange;
 
 		if (realHealthChange > 0 && !target->isInGhostMode()) {
-			std::stringstream ss;
-
-			ss << realHealthChange << (realHealthChange != 1 ? " hitpoints." : " hitpoint.");
-			std::string damageString = ss.str();
-
-			std::string spectatorMessage;
+			auto damageString = fmt::format("{:d} hitpoint{:s}.", realHealthChange, realHealthChange != 1 ? "s" : "");
```

Comment:
![image](https://user-images.githubusercontent.com/28090948/109369666-974e5f00-7873-11eb-9d56-ceb7f2b50c27.png)
this point is out of place.

Reply:
Oops.